src/js/oErrors.js

```diff
@@ -4,6 +4,10 @@ function isFunction(fn) {
 	return typeof fn === 'function';
 }
 
+function isArray(arr) {
```

Comment:
think there's an additional step… to declare the dependency on this polyfill in the `origami.json` file